packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -135,11 +136,7 @@ export const ExpandableBlock = (props: ExpandableBlockProps) => {
           <div className='iui-title'>{title}</div>
           {caption && <div className='iui-caption'>{caption}</div>}
         </span>
-        {icon &&
-          React.cloneElement(icon, {
-            className: cx('iui-status-icon', icon.props.className),
-            'data-iui-icon-color': status,
-          })}
+        {icon && <Icon fill={status}>{icon}</Icon>}
```

Comment:
We have several places in react package where we are wrapping icon with span (eg. Button start and end icons). Does it makes sense to also update those with new component or is it breaking change?

Reply:
we cannot remove the old class names but we could support both use cases. i'm not sure of the benefit though